control/runtimes/rococo/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "rococo-runtime"
```

Comment:
Will we ever publish these crates? https://crates.io/crates/rococo-runtime If we will, maybe we should prefix with snowbridge something.

Reply:
Nah, I think we will only compile this tool on-demand.